### ISSUE TITLE
Training loss suddenly decrease at the beginning of each epoch.

### ISSUE THREAD

**Issue by Tebmer** [thu-coai/Emotional-Support-Conversation#14]
Hi,

I use the code in `codes` to train model. But I find a strange phenomenon that the loss suddenly decreases at the beginning of every epoch. 

So, my question is that is this the same as your training curve? and why does it suddenly decrease at the beginning of every epoch? I thought about it for a long time, but I still can't figure it out. Thanks a lot!
<img width="353" alt="image" src="https://user-images.githubusercontent.com/37136730/175063527-2fb2aa26-fcfe-4f65-9569-66e38d091c93.png">

**Comment by lsy641** [comment]
which version of code do you use?

> Hi,
> 
> I use the code in `codes` to train model. But I find a strange phenomenon that the loss suddenly decreases at the beginning of every epoch.
> 
> So, my question is that is this the same as your training curve? and why does it suddenly decrease at the beginning of every epoch? I thought about it for a long time, but I still can't figure it out. Thanks a lot! <img alt="image" width="353" src="https://user-images.githubusercontent.com/37136730/175063527-2fb2aa26-fcfe-4f65-9569-66e38d091c93.png">

**Comment by Tebmer** [comment]
The first one, `codes` directory.

**Comment by Tebmer** [comment]
@lsy641 Hi, I use the first one version code, i.e. `codes` directory,

**Comment by lsy641** [comment]
Sorry, I didn't see the newest comment. I didn't notice a strange curve when training. BTW I fixed a bug in June 25. @Tebmer
